src/main/java/com/vinylteam/vinyl/util/PropertiesReader.java

```diff
@@ -16,10 +16,9 @@
     private final String independentPropertiesFile = "application.properties";
 
     public PropertiesReader() {
-        String beginningOfErrorMessage = "Error during loading properties from";
-
+        String beginningOfErrorMessage = "Error during loading properties from ";
         try (InputStream inputStream = getClass().getClassLoader()
-                .getResourceAsStream(independentPropertiesFile)) {
+                .getResourceAsStream("application.properties")) {
             logger.debug("Created input stream from file {'inputStream':{}, 'fileName':{}}",
                     inputStream, independentPropertiesFile);
```

Comment:
Decide if you use independentPropertiesFile string or just use "application.properties" everywhere